conf/openmetadata.yaml

```diff
@@ -256,7 +256,8 @@ elasticsearch:
   socketTimeoutSecs: ${ELASTICSEARCH_SOCKET_TIMEOUT_SECS:-60}
   keepAliveTimeoutSecs: ${ELASTICSEARCH_KEEP_ALIVE_TIMEOUT_SECS:-600}
   batchSize: ${ELASTICSEARCH_BATCH_SIZE:-10}
-  searchIndexMappingLanguage: ${ELASTICSEARCH_INDEX_MAPPING_LANG:-EN}
+  searchIndexMappingLanguage : ${ELASTICSEARCH_INDEX_MAPPING_LANG:-EN}
+  searchIndexFactoryClassName : org.openmetadata.service.search.SearchIndexFactory
```

Comment:
any reason we need this as config?

Reply:
@harshach we have two index factory one is SearchIndexFactory and one is CollateIndexFactory so we are making that object from classname.